### DISCUSSION TITLE
Vertical photos are rotated horizontally

### DISCUSSION THREAD

**Discussion by giuenrico93** [opengisch/QField#4286] [Bugs]
Hi
I have the following problem: if in QFieldSync I set a maximum size for the longest side of the photo, all photos taken vertically, using QField Cloud, are rotated horizontally. 
How can I solve this? I use a smartphone with Android 12 (Motorola edge 20 pro). 
Thank you!

**Comment by aliasatp** [comment]
I have the same issue. It has appeared again in last qfield releases. In fact it was solved but now it happens again.
